include/cantera/oneD/Flow1D.h

```diff
@@ -849,48 +891,99 @@ class Flow1D : public Domain1D
 
     size_t m_nsp; //!< Number of species in the mechanism
 
+    //! Phase object used for calculating thermodynamic properties
     ThermoPhase* m_thermo = nullptr;
+
+    //! Kinetics object used for calculating species production rates
     Kinetics* m_kin = nullptr;
+
+    //! Transport object used for calculating transport properties
     Transport* m_trans = nullptr;
 
-    // boundary emissivities for the radiation calculations
+    //! Emissivity of the surface to the left of the domain. Used for calculating
+    //! radiative heat loss.
     double m_epsilon_left = 0.0;
+
+    //! Emissivity of the surface to the right of the domain. Used for calculating
+    //! radiative heat loss.
     double m_epsilon_right = 0.0;
 
     //! Indices within the ThermoPhase of the radiating species. First index is
     //! for CO2, second is for H2O.
     vector<size_t> m_kRadiating;
 
-    // flags
+    //! @name flags
+    //! @{
+
+    //! For each point in the domain, `true` if energy equation is solved or `false` if
+    //! temperature is held constant.
+    //! @see doEnergy, fixTemperature
     vector<bool> m_do_energy;
+
+    //! `true` if the Soret diffusion term should be calculated.
     bool m_do_soret = false;
+
+    //! Determines whether diffusive fluxes are computed using gradients of mass
+    //! fraction or mole fraction.
+    //! @see setFluxGradientBasis, fluxGradientBasis
     ThermoBasis m_fluxGradientBasis = ThermoBasis::molar;
-    vector<bool> m_do_species;
```

Comment:
It would be good if you could comment on the rationale for removing this feature.

Reply:
The feature was never implemented. This was just a variable that had no methods for being set except when being restored from a saved data file, and was never used to control the logic for evaluating the governing equations.